db/error-handler.js

```diff
@@ -19,6 +18,9 @@ function errorHandler(err, req, res, next) {
     else if (err === 'Error: 404 - Not Found'){
         res.status(404).send({ Error: 'ID Does Not Exist' })
     }
+    else if (err === 'Error: 200 - No Articles'){
+        res.status(200).send({articles: []})
+    }
```

Comment:
Do you really want a error with a 200 status code? Would that be what you would expect from an error? 🧐